pkg/cmd/cli/backup/logs.go

```diff
@@ -46,15 +50,63 @@ func NewLogsCommand(f client.Factory) *cobra.Command {
 			arkClient, err := f.Client()
 			cmd.CheckError(err)
 
-			backup, err := arkClient.ArkV1().Backups(v1.DefaultNamespace).Get(args[0], metav1.GetOptions{})
+			_, err = arkClient.ArkV1().Backups(v1.DefaultNamespace).Get(args[0], metav1.GetOptions{})
+			cmd.CheckError(err)
+
+			dr := &v1.DownloadRequest{
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: v1.DefaultNamespace,
+					Name:      fmt.Sprintf("%s-%s", args[0], time.Now().Format("20060102150405")),
+				},
+				Spec: v1.DownloadRequestSpec{
+					Target: v1.DownloadTarget{
+						Kind: v1.DownloadTargetKindBackupLog,
+						Name: args[0],
+					},
+				},
+			}
+
+			dr, err = arkClient.ArkV1().DownloadRequests(v1.DefaultNamespace).Create(dr)
+			cmd.CheckError(err)
+
+			listOptions := metav1.ListOptions{
+				//TODO: once kube-apiserver http://issue.k8s.io/51046 is fixed, uncomment
+				//FieldSelector:   "metadata.name=" + args[0],
```

Comment:
Yes, thanks. I also need to make sure the name of the DownloadRequest received from w.ResultChan matches that too.